engine/polyglot-api/src/main/java/org/enso/polyglot/RuntimeOptions.java

```diff
@@ -29,13 +29,20 @@
   private static final OptionDescriptor LOG_LEVEL_DESCRIPTOR =
       OptionDescriptor.newBuilder(LOG_LEVEL_KEY, LOG_LEVEL).build();
 
+  public static final String DISABLE_PARALLEL_EXECUTION = optionName(".disableParallelExecution");
```

Comment:
Please change the name to something indicating this is language server specific, currently looks like "this will disable threading in the language".